blackbox/docs/appendices/release-notes/3.2.0.rst

```diff
@@ -0,0 +1,198 @@
+     which is ``@deprecated``.
+
+   - Addition of French language files and menu options.
+
+   - Fixed an issue that caused incorrectness in the navigation of the Admin UI.
+
+   - Updated the license div to be responsive in the Admin UI.
+
+   - Various Admin UI improvements.
+
+- Changed PostgreSQL wire interface to emulate version ``10.5``.
+
+-  Added support for CrateDB license management enabling users to trial the
+   enterprise features, set a production enterprise license or continue
+   using the community edition. Additionally, a new ``SET LICENSE`` statement
+   has been added for license registration, and the ``license.ident`` setting
+   has become ``@deprecated``.
+
+- The ``MQTT`` endpoint has been deprecated and will be removed in a future
+  version.
+
+- Upgraded to Elasticsearch 6.5.1, which includes changes to the default logging
+  configuration.
+
+- Deprecated the ``http.enabled`` setting.
+
+- Added a ``remove_duplicates`` token filter.
+
+- Added a ``char_group`` tokenizer.
+
+- Renamed the ``delimited_payload_filter`` token filter to
+  ``delimited_payload``. The old name can still be used, but is deprecated.
+
+- Added the ``GENERATE_SERIES(start, stop [, step ])`` table function.
+
+- Added ``pg_class``, ``pg_namespace``, ``pg_attribute``, ``pg_attrdef``,
+  ``pg_index`` and ``pg_constraint`` tables to the ``pg_catalog`` schema for
+  improved compatibility with postgresql.
+
+- Added support for using table functions in the select list of a query.
+
+- Implemented the ``array_upper``, ``array_length`` and ``array_lower`` scalars
+  that return the upper and respectively lower bound of a given array
+  dimension.
+
+- Added some type aliases for improved compatibility with postgresql.
+
+- Expand the ``search_path`` setting to accept a list of schemas that will be
+  searched when a relation (table, view or user defined function) is referenced
+  without specifying a schema. The system ``pg_catalog`` schema is implicitly
+  included as the first one in the path.
+
+- Improved the handling of function expressions inside subscripts used on
+  object columns. This allows expressions like ``obj['x' || 'x']`` to be used.
+
+- The ``= ANY`` operator now also supports operations on object arrays or
+  nested arrays. This enables queries like ``WHERE ['foo', 'bar'] =
+  ANY(object_array(string_array))``.
+
+- Added support for the ``ARRAY(subquery)`` expression.
+
+- ``<object_column> = <object_literal>`` comparisons now try to utilize the
+  index for the objects contents and can therefore run much faster.
+
+- Values of byte-size and time based configuration setting do not require a unit
+  suffix anymore. Without a unit time values are treat as milliseconds since
+  epoch and byte size values are treat as bytes.
+
+- Added support of using units inside byte-size or time bases statement
+  parameters values. E.g. '1mb' for 1 MegaByte or '1s' for 1 Second.
+
+.. _version_3.2.0_upgrade_notes:
+
+Upgrade Notes
```

Comment:
I'd put these before the list of changes, but no strong opinion.

Reply:
Yeah, I think I'd agree.